pkg/rbd/rbd_util.go

```diff
@@ -67,6 +67,7 @@ type rbdVolume struct {
 	VolID              string `json:"volID"`
 	Monitors           string `json:"monitors"`
 	Pool               string `json:"pool"`
+	DataPool           string `json:"dataPool"`
```

Comment:
(nit) I removed json annotations from elements of the structure that were not stores in the config maps in v1.0 (and earlier) of the code base. Going by the same yardstick, remove the trailing json annotation on this field.

Reply:
forgot about 1.0.0 configmaps